core/bundle/changes/export_test.go

```diff
@@ -0,0 +1,17 @@
+// Copyright 2021 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package bundlechanges
```

Comment:
This should go in _package_test.go_. This package is recent enough to have been set up properly with a package-local package_test.